### DISCUSSION TITLE
Function to string

### DISCUSSION THREAD

**Discussion by JanGoos-Engibex** [casadi/casadi#3679] [Q&A] [answered]
Hi all

Following the [documentation](https://web.casadi.org/docs/#document-function) 
```
x = SX.sym('x',2);
y = SX.sym('y');
f = Function('f',[x,y],[x,sin(y)*x]);
disp(f)
```
gives me a 
```
f:(i0[2],i1)->(o0[2],o1[2]) SXFunction
```
Can I get a string print of a Function e.g. f[1] = sin(y)*x or something?

Cheers
J a n

**Comment by JanGoos-Engibex** [comment, marked as answer]
After some digging, I found I can call 
```
 f.call([x,y])
```
resulting in
```
[SX([x_0, x_1]), SX(@1=sin(y), [(@1*x_0), (@1*x_1)])]
```
As stated in [4.1 Calling function objects](https://web.casadi.org/docs/#calling-function-objects).
I only saw the numerical call, my bad.

**Reply by nashmit** [reply]
One can also do: 
`f(x,y)`
You get the same result.
